code/game/area/areas.dm

```diff
@@ -145,46 +145,33 @@
 	if(!eject)
 		eject = 1
 		updateicon()
-	return
 
 /area/proc/readyreset()
 	if(eject)
 		eject = 0
 		updateicon()
-	return
 
 /area/proc/radiation_alert()
 	if(!radalert)
 		radalert = 1
 		updateicon()
-	return
 
 /area/proc/reset_radiation_alert()
 	if(radalert)
 		radalert = 0
 		updateicon()
-	return
 
 /area/proc/partyalert()
-	if(!( party ))
+	if(!party)
 		party = 1
 		updateicon()
 		mouse_opacity = 0
-	return
 
 /area/proc/partyreset()
 	if(party)
 		party = 0
 		mouse_opacity = 0
 		updateicon()
-		for(var/obj/machinery/door/firedoor/D in src)

```

Comment:
Not sure why this opened the fire doors, because the alert itself never closed them.